docs/src/components/programs.md

```diff
@@ -0,0 +1,11 @@
+# Programs
+
+- **Valence Program:** A program is an instance of the Valence Protocol. It is a particular arrangement and configuration of _ValenceAccounts_ and _Function_ across multiple _domains_. Programs are also associated with a set of _Subroutines_ that can be executed on the _Program_. For example, a POL lending relationship between two parties may be set up as a program.
```

Comment:
Shouldn't it be: `It is a particular arrangement and configuration of _ValenceAccounts_ and _Libraries_ across multiple _domains_`

Reply:
Correct, fixed it.